Lesson-7/style.css

```diff
@@ -0,0 +1,1071 @@
+        width: 320px;
+        }
+        .text_headphones {
+        font-family: Roboto;
+        font-style: normal;
+        font-weight: normal;
+        font-size: 15px;
+        line-height: 18px;
+        text-align: center;
+        color: #383838;
+        margin-bottom: 47px;
+        }
+        .text_communicate {
+        font-family: Roboto;
+        font-style: normal;
+        font-weight: normal;
+        font-size: 15px;
+        line-height: 18px;
+        text-align: center;
+        color: #383838;
+        margin-bottom: 43px;
+        }    
+        .text_gadgets {
+        font-family: Roboto;
+        font-style: normal;
+        font-weight: normal;
+        font-size: 15px;
+        line-height: 18px;
+        text-align: center;
+        color: #383838;
+        margin-bottom: 34px;
+        }   
+        .text_like {
+        font-family: Roboto;
+        font-style: normal;
+        font-weight: normal;
+        font-size: 15px;
+        line-height: 18px;
+        text-align: center;
+        color: #383838;
+        align-items: center;
+    
+        }
+        .headline {
+        font-style: normal;
+        font-weight: 500;
+        font-size: 28px;
+        line-height: 33px;
+        color: #383838;
+        text-align:center;
+        display: flex;
+        justify-content: center;
+        width: 320px;
+        }
+        .headphones {
+        height: 64px;
+        width: 53px;
+        margin-bottom: 35px;
+        }
+        .communicate {
+        width: 58px;
+        height: 58px;
+        margin-bottom: 35px;
+        }
+        .gadgets {
+        width: 70px;
+        height: 47px;
+        margin-bottom: 35px;
+        }
+        .like_img {
+        width: 64px;
+        height: 58px;
+        margin-bottom: 35px;
+        }
+        .container_headphones {
+        margin-bottom: 35px;
+        width: 214px;
+        height: 104px;
+        }
+        .container_communicate {
+        margin-bottom: 35px;
+        width: 214px;
+        height: 104px;
+        }
+        .container_gadgets {
+        margin-bottom: 35px;
+        width: 214px;
+        height: 104px;
+        }
+        .container_like {
+        margin-bottom: 35px;
+        width: 214px;
+        height: 104px;
+        }
+        .container_heading {
+        display: flex;
+        justify-content: center;
+        flex-direction: column;
+        align-items: center;
+        width: 360px;
+    
+        }
+        .top {
+        display: none;
+        }
+        .heading_about {
+        text-decoration: none;
+        }
+        .heading_about:hover {
+        text-decoration: underline;
+        }
+        .heading_property {
+        text-decoration: none;
+        }
+        .heading_property:hover {
+        text-decoration: underline;
+        }
+        .heading_support {
+        text-decoration: none;
+        }
+        .heading_support:hover {
+        text-decoration: underline;
+        }
```

Comment:
these styles contain the same thing. they can be separated by commas. BUT here it should be used only one class.